test/functionalTest/cases/1308_subscription_cache/subCacheMixedSubscriptionsWithUpdatesAndDeletes.test

```diff
@@ -0,0 +1,581 @@
+# Copyright 2015 Telefonica InvesTigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Subscription Cache Mixed wildcard and not and removes and updates
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 101
+accumulatorStart
+
+--SHELL--
+
+#
+# 01. Create subscription S1 - without pattern
```

Comment:
I have looked the test "script" and seem to be ok (I haven't reviewed the details of the test itself).

NTC (just informative)